tree/dataframe/src/RDataFrame.cxx

```diff
@@ -620,7 +645,7 @@ RDataFrame operations currently work with this package. The subset that is curre
 - Fill
 - Filter
 - Graph
-- Histo[1,2,3]D
+- Histo[1,2,3,N]D
 - HistoND
```

Comment:
then next line should go

Reply:
I actually didn't read it, I think it is fine as it it, in the latest commit I removed this addition

Reply:
Thanks for catching it!